relay-general/src/protocol/contexts/device.rs

```diff
@@ -174,9 +174,14 @@ impl DeviceContext {
 
 impl FromUserAgentInfo for DeviceContext {
     fn parse_client_hints(client_hints: &ClientHints<&str>) -> Option<Self> {
-        let device = client_hints.sec_ch_ua_model?.to_owned();
-
-        if device.trim().is_empty() {
+        let device = client_hints
+            .sec_ch_ua_model?
+            .trim()
+            .trim_start_matches('\"')
+            .trim_end_matches('\"')
+            .to_owned();
```

Comment:
Why just do not use [`replace`](https://doc.rust-lang.org/std/primitive.str.html#method.replace) instead all of this?

Reply:
I didn't think of that, I changed it now